governance/votes/Executive vote - April 1, 2022.md

```diff
@@ -0,0 +1,90 @@
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Core Unit Budget Transfers
+
+As per multiple successful budget proposals (linked below), a total of **2,433,452.30 DAI** will be transferred to multiple Core Units if this executive proposal passes:
+
+- 30,000 DAI will be transferred to the [GovAlpha Core Unit (GOV-001)](https://mips.makerdao.com/mips/details/MIP39c2SP3) at [0x01D26f8c5cC009868A4BF66E268c17B057fF7A73](https://etherscan.io/address/0x01D26f8c5cC009868A4BF66E268c17B057fF7A73) as a result of [their most recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP59).
+- 348,452.30 DAI will be transferred to the [Immunefi Security Core Unit (IS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP24) at [0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4](https://etherscan.io/address/0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4) as a result of [an amendment to their MIP40c3-SP42 budget](https://mips.makerdao.com/mips/details/MIP4c2SP17).
+- 2,055,000 DAI will be transferred to the [Real-World Finance Core Unit (RWF-001](https://mips.makerdao.com/mips/details/MIP39c2SP1) at [0x96d7b01Cc25B141520C717fa369844d34FF116ec](https://etherscan.io/address/0x96d7b01Cc25B141520C717fa369844d34FF116ec) as a result of [their most recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP61).
+
+
+### Core Unit Budget Streams
+
+As per multiple successful budget proposals (linked below), a total of **7,945,149.90 DAI** will be streamed to multiple Core Units if this executive proposal passes:
+
+- 1,079,793 DAI will be streamed to the [GovAlpha Core Unit (GOV-001)](https://mips.makerdao.com/mips/details/MIP39c2SP3) at [0x01D26f8c5cC009868A4BF66E268c17B057fF7A73](https://etherscan.io/address/0x01D26f8c5cC009868A4BF66E268c17B057fF7A73) starting on 2022-04-01 and ending on 2023-04-01 as a result of [their most recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP59).
+- 700,356.90 DAI will be streamed to the [Immunefi Security Core Unit (IS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP24) at [0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4](https://etherscan.io/address/0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4) starting on 2022-03-01 and ending on 2022-08-01 as a result of [an amendment to their MIP40c3-SP42 budget proposal](https://mips.makerdao.com/mips/details/MIP4c2SP17).
+- 6,165,000 DAI will be streamed to the [Real-World Finance Core Unit (RWF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP1) at [0x96d7b01Cc25B141520C717fa369844d34FF116ec](https://etherscan.io/address/0x96d7b01Cc25B141520C717fa369844d34FF116ec) starting on 2022-04-01 and ending on 2022-12-31 as a result of [their most recent budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP61).
```

Comment:
Why are the DAI amounts in this section not in **bold** while the amounts lower down are?